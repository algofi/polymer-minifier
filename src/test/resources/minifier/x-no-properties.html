
<dom-module id="x-no-properties">
	<template>
		<style>
		</style>
		<!-- this is a comment to remove -->
		<!-- 
			this is a longer comment
			this is a longer comment
			this is a longer comment
			this is a longer comment
			this is a longer comment
			this is a longer comment
			this is a longer comment
		 -->
		<script>
			Polymer( {
				is:'x-no-properties'
			} );
		</script>
	</template>
</dom-module>