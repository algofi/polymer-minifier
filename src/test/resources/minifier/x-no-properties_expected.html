<dom-module id="x-no-properties"><template><style></style><script>
			Polymer( {
				is:'x-no-properties'
			} );
		</script></template></dom-module>